### ISSUE TITLE
Cannot use `aldeed:schema-index`, getting error "index" is not a supported property

### ISSUE THREAD

**Issue by nicu-chiciuc** [ardatan/meteor-webpack#41]
I'm having this issue with `aldeed:schema-index` although I think the same issue would apply also to other meteor packages. Basically what this package does is add several options to SimpleSchema.
```
SimpleSchema.extendOptions([
  'index', // one of Number, String, Boolean
  'unique', // Boolean
  'sparse', // Boolean
]);
```

The problem is that I'm getting the following error:
```
Error: Invalid definition for username field: "index" is not a supported property
```

I've tried to use your example of React by adding `simple-schema` and I'm getting the exact same error.
It seems like the meteor package is not executed or is using a different version of SimpleSchema.

Or maybe I missed something in how to actually use this package since I don't have much experience in meteor.

**Comment by ardatan** [comment]
Does this work well without webpack?

**Comment by nicu-chiciuc** [comment]
Yes, I think it might have something to do with the fact that [`simpl-schema`](https://github.com/aldeed/simple-schema-js#installation) is installed using npm and the [aldeed:schema-index](https://github.com/aldeed/meteor-schema-index) is installed using atmosphere and it basically extends the package from npm.

[schema-index  common.js](https://github.com/aldeed/meteor-schema-index/blob/master/package/indexing/common.js)
```
// collection2 checks to make sure that simpl-schema package is added
import SimpleSchema from 'simpl-schema';
import Collection2 from 'meteor/aldeed:collection2';

// Extend the schema options allowed by SimpleSchema
SimpleSchema.extendOptions([
  'index', // one of Number, String, Boolean
  'unique', // Boolean
  'sparse', // Boolean
]);

Collection2.on('schema.attached', (collection, ss) => {
  // Define validation error messages
  if (ss.version >= 2 && ss.messageBox && typeof ss.messageBox.messages === 'function') {
    ss.messageBox.messages({
      en: {
        notUnique: '{{label}} must be unique',
      },
    });
  }
});
```

I'll probably have to figure out how the code in `meteor-webpack` manages stuff like this, but maybe you'd have some tips.

**Comment by pmogollons** [comment]
I just had this issue too. It might be possible that schema-index loads first and the simpl-schema gets loaded and reconfigured somehow and that removes the extended options.

Im going to try to add package functionality in app code and check if that works.

@nicu-chiciuc where you able to find a solution?

**Comment by pmogollons** [comment]
I was able to workaround the schema-index issue by not importing the schemas in the client code.

**Comment by ardatan** [comment]
Can you share your webpack configuration?
